README.md

```diff
@@ -172,11 +206,17 @@ The HTML element where the manager will build the steps.
 
 `Object`
 
-JSON datas for all steps.
+The datas for all the steps stored in a JSON. The object key need to match with the route id declare in each steps.
```

Comment:
_The datas for all the steps are stored in a JSON. The object key needs to match with the route id declared in each step._

Reply:
C'est corrigé